installation/proc-install-rhdh-ocp-operator.adoc

```diff
@@ -1,28 +1,37 @@
-[id='proc-install-rhdh-ocp-operator_{context}']
-
+[id="proc-install-rhdh-ocp-operator_{context}"]
 = Deploying {product} on {ocp-short} using the Operator
 
-As a developer, you can install {product} on your OpenShift Container Platform using the {product} Operator.
+As a developer, you can deploy a {product} instance on {ocp-short} by using the *Developer* perspective in the web console.
 
 .Prerequisites
-* Your administrator has installed the {product} Operator. For more information, see xref:proc-install-operator_admin-rhdh[Installing the {product} Operator].
-* You have created a project in {ocp-short} for your {product} instance. For more information about creating a project in OpenShift, see link:https://docs.openshift.com/container-platform/4.14/applications/projects/working-with-projects.html#odc-creating-projects-using-developer-perspective_projects[Red Hat OpenShift documentation].
+
+* A cluster administrator has installed the {product} Operator. For more information, see xref:proc-install-operator_admin-rhdh[Installing the {product} Operator].
 
 .Procedure
-. From the *Developer* perspective on the {product-short} web console, click *+Add*.
+
+. Create a project in {ocp-short} for your {product} instance, or select an existing project.
++
+[TIP]
+====
+For more information about creating a project in {ocp-short}, see link:https://docs.openshift.com/container-platform/4.15/applications/projects/working-with-projects.html#creating-a-project-using-the-web-console_projects[Creating a project by using the web console] in the {ocp-brand-name} documentation.
+====
+. From the *Developer* perspective on the {ocp-short} web console, click *+Add*.
 . From the *Developer Catalog* panel, click *Operator Backed*.
```

Comment:
I am wondering if it would make sense to squeeze this step with the previous.

Something like:

From the *+Add* tab in the {ocp-short} web console *Developer* perspective, in the *Developer Catalog* panel, click *Operator Backed*.

Reply:
I'd prefer to leave this as is for now since this step wasn't part of the scope of the original PR/Jira.